web_external/Literature/literaturePage.styl

```diff
@@ -0,0 +1,22 @@
+.isic-literature
+  table
+    width 1170px
+
+    thead
+      border-style solid
+      border-width 2px 0
+      tr
+        th.isic-literature-column-year
+          width 60px
+        th.isic-literature-column-type
+          width 110px
+    tbody
+      tr
+        &:nth-child(even)
+          background-color #eeeeee
+        &:nth-child(odd)
+          background-color #ffffff
```

Comment:
Bootstrap has `.table-striped`, but if it doesn't do exactly what you want out of the box you'll probably end up using rules like this anyway.